crates/debug/src/transform/address_transform.rs

```diff
@@ -678,11 +689,12 @@ mod tests {
     fn test_build_function_lookup_two_ranges() {
         let mut input = create_simple_func(11);
         // append instruction with same srcloc as input.instructions[0]
-        input.instructions.push(InstructionAddressMap {
+        let mut list = Vec::from(mem::take(&mut input.instructions));
+        list.push(InstructionAddressMap {
             srcloc: SourceLoc::new(11 + 2),
             code_offset: 23,
-            code_len: 3,
```

Comment:
Should this have a trailing default range starting at 26 so that the below assert (e.g. `assert_eq!(26, positions[0].gen_end);`) remains unchanged?

Reply:
Makes sense to me!